config/storybook/webpack.config.js

```diff
@@ -6,11 +6,20 @@ const nuxtRootDir = path.resolve(`${__dirname}/../../src/client`)
 module.exports = (baseConfig, env, defaultConfig) => {
   const isDev = (env === 'DEVELOPMENT')
   const isClient = true
+  const urlLoader = defaultConfig.module.rules.find((rule) => /file-loader/.test(rule.loader))
```

Comment:
I'm happy to have this loader in the project. However I would still try to load SVGs as regular images as much as possible, as when they are inlined they are served as part of the JS file. Separate images are easier to cache and keep our JS bundles smaller.